text/0000-landmarks.md

```diff
@@ -0,0 +1,29 @@
+# Landmark observations
+Add landmarks support to Cartographer.
+
+## Summary
+[summary]: #summary
+Cartographer will be able to work with a new type of sensor data that contains IDs and locations of landmarks.
+
+## Motivation
+[motivation]: #motivation
+Landmarks translation/orientation can improve SLAM quality. Landmark visualizations can be useful for inspection.
+
+## Approach
+[approach]: #approach
+Landmarks are sparse features of geometry that we want to map. Unlike dense range data, landmarks are assumed to be sparse in comparison. In order to support using them, current SLAM approach has to be generalized. 
+
+### Sensor
+Each landmark will have an ID and a position relative to the robot with or without orientation. A landmark observation can contain several landmarks at a time.
```

Comment:
It is also important to allow "observations" with zero landmarks seen, so that SLAM can continue, right?

Reply:
Yes, that's why we are sending a vector of landmarks that can be empty.

Reply:
Could you make this a bit clearer in the RFC? How about
```
Each landmark observation contains all landmarks for which a relative position is currently observed. This can be an empty list.
```
?

Reply:
Done